runtime/include/chpl-mem.h

```diff
@@ -81,6 +81,21 @@ void* chpl_mem_alloc(size_t size, chpl_mem_descInt_t description,
   return chpl_mem_allocMany(1, size, description, lineno, filename);
 }
 
+// assumes alignment is a power of 2
+// assumes that size is a multiple of alignment
+static inline
+void* chpl_mem_aligned_alloc(size_t alignment, size_t size,
```

Comment:
We have chpl_mem_memalign. I think we should just call that

Reply:
I think you're pointing out that this new function is duplicative with chpl_mem_memalign. However if we are renaming chpl_memalign then we would presumably "rename" chpl_mem_memalign to chpl_mem_aligned_alloc, right?